import_export/widgets.py

```diff
@@ -213,40 +213,47 @@ def render(self, value, obj=None):
         return self.TRUE_VALUES[0] if value else self.FALSE_VALUES[0]
 
 
-class DateWidget(Widget):
+class BaseDateTimeWidget(Widget):
```

Comment:
I wonder if we should make this class abstract, since it should never be instantiated.